nixos/modules/virtualisation/podman/default.nix

```diff
@@ -5,7 +5,7 @@ let
 
   inherit (lib) mkOption types;
 
-  podmanPackage = (pkgs.podman.override {
+  podmanPackage = ((cfg.package or pkgs.podman).override {
```

Comment:
`cfg.package` is already Podman by default, or was this required to get it to eval?

```suggestion
  podmanPackage = (cfg.packageoverride {
```

Reply:
I'm not really sure where `cfg.packageoverride` comes from. Is this generated somehow?

Reply:
Typo, sorry. Edited, does it make sense now?

Reply:
Makes total sense! Thanks for the suggestion